Master broken on CI

Here's a simple CI run on a PR that doesn't change anything (just javadoc change to trigger CI)

https://jenkins.konduit.ai/blue/organizations/jenkins/konduitai%2Fkonduit-serving/detail/PR-273/1/pipeline/79

https://github.com/KonduitAI/konduit-serving/pull/273

```
[2020-03-23T12:29:06.508Z] [ERROR]   TestPythonSetupRun.testInferenceResult:118 2 expectations failed.
[2020-03-23T12:29:06.508Z] Expected status code <200> doesn't match actual status code <500>.
[2020-03-23T12:29:06.508Z] 
[2020-03-23T12:29:06.508Z] Response body doesn't match expectation.
[2020-03-23T12:29:06.508Z] Expected: not (null or an empty string)
[2020-03-23T12:29:06.508Z]   Actual: 
[2020-03-23T12:29:06.508Z] 
[2020-03-23T12:29:06.508Z] [ERROR] Errors: 
[2020-03-23T12:29:06.508Z] [ERROR]   PipelineExecutionerTests.testInitPipeline:207 ? IllegalState Unable to instant...
[2020-03-23T12:29:06.508Z] [ERROR]   PipelineExecutionerTests.testInitPipelineYolo:260 ? IllegalState Unable to ins...
[2020-03-23T12:29:06.508Z] [ERROR]   TensorflowInferenceExecutionerTests.testInferenceExecutioner:58 ? NoClassDefFound
[2020-03-23T12:29:06.508Z] [ERROR]   TensorflowInferenceExecutionerTests.testSavedModelInferenceExecutioner:88 ? NoClassDefFound
[2020-03-23T12:29:06.508Z] [ERROR]   InferenceExecutionerFactoryTests.testTensorflow:74 ? NoClassDefFound Could not...
```

--- a/konduit-serving-api/src/main/java/ai/konduit/serving/util/SchemaTypeUtils.java
+++ b/konduit-serving-api/src/main/java/ai/konduit/serving/util/SchemaTypeUtils.java
@@ -38,7 +38,6 @@ import org.datavec.api.writable.Writable;
 import org.datavec.image.data.ImageWritable;
 import org.nd4j.base.Preconditions;
 import org.nd4j.linalg.api.ndarray.INDArray;
-import org.nd4j.linalg.factory.Nd4j;
 
 import java.util.*;
 

--- a/konduit-serving-tfjava/pom.xml
+++ b/konduit-serving-tfjava/pom.xml
@@ -13,29 +13,19 @@
 
     <name>konduit-serving-tfjava</name>
 
-
     <dependencies>
         <dependency>
             <groupId>org.bytedeco</groupId>
             <artifactId>tensorflow</artifactId>
-            <version>${tensorflow.javacpp.version}</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.bytedeco</groupId>
-                    <artifactId>mkl-dnn</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.bytedeco</groupId>
-                    <artifactId>mkl-dnn-platform</artifactId>
-                </exclusion>
-            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.bytedeco</groupId>
+            <artifactId>tensorflow-platform</artifactId>
         </dependency>
         <dependency>
             <groupId>org.nd4j</groupId>
             <artifactId>nd4j-tensorflow</artifactId>
-            <version>${datavec.version}</version>
         </dependency>
-
         <dependency>
             <groupId>ai.konduit.serving</groupId>
             <artifactId>konduit-serving-core</artifactId>

--- a/pom.xml
+++ b/pom.xml
@@ -355,16 +355,6 @@
             </dependency>
             <dependency>
                 <groupId>org.bytedeco</groupId>
-                <artifactId>tensorflow-platform</artifactId>
-                <version>${tensorflow.javacpp.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.bytedeco</groupId>
-                <artifactId>tensorflow</artifactId>
-                <version>${tensorflow.javacpp.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.bytedeco</groupId>
                 <artifactId>javacv</artifactId>
                 <version>${javacpp.version}</version>
             </dependency>
@@ -407,6 +397,26 @@
                 <groupId>org.nd4j</groupId>
                 <artifactId>nd4j-jackson</artifactId>
                 <version>${dl4j.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.bytedeco</groupId>
+                <artifactId>tensorflow</artifactId>
+                <version>${tensorflow.javacpp.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.bytedeco</groupId>
+                        <artifactId>mkl-dnn</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.bytedeco</groupId>
+                        <artifactId>mkl-dnn-platform</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>org.bytedeco</groupId>
+                <artifactId>tensorflow-platform</artifactId>
+                <version>${tensorflow.javacpp.version}</version>
             </dependency>
             <dependency>
                 <groupId>org.nd4j</groupId>

